src/components/Login.js

```diff
@@ -2,15 +2,8 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { postLogin } from '../actions/login';
-import {
-  Form,
-  Grid,
-  Image,
-  Divider,
-  Icon,
-  Message,
-  Button,
-} from 'semantic-ui-react';
+import { Form, Image, Divider, Icon, Message, Button } from 'semantic-ui-react';
+import { Container } from 'react-bootstrap';
```

Comment:
You have imported here `{ Container } from 'react-bootstrap'` but `react-bootstrap` is not in package.json. It is not compiling locally.
Also container is present in semantic-ui: https://react.semantic-ui.com/elements/container/

Reply:
Okay, I haven't used semantic-ui, So let me see how responsive containers work in semantic-ui. Btw can I use bootstrap classes to make containers responsive?